tests/test_utils.py

```diff
@@ -377,3 +388,34 @@ def test_load_spacy_model(model_name, is_found):
     else:
         with pytest.raises(ModuleNotFoundError):
             load_spacy_model(model_name)
+
+
+@pytest.mark.parametrize(
+    "get_obj",
+    [
+        lambda: torch.tensor([8.0, 8.0, 5.0]),
+        lambda: {
+            "a": torch.tensor([8.0, 8.0, 5.0]),
+            "b": torch.tensor([3.0, 1.0, 2.0, 4.0]),
+            "c": torch.tensor([[1.0, 1.9], [1.2, 1.3]]),
+        },
+        lambda: Sequential(Linear(3, 5), Linear(5, 2)),
+    ],
```

Comment:
I would add one more test case - state dictionary of a module. It is the recommended way how to save models for inference (according to https://pytorch.org/tutorials/beginner/saving_loading_models.html#saving-loading-model-for-inference)

So something like 
```python
lambda: Sequential(Linear(3, 5), Linear(5, 2)).state_dict()
```
It seems to use `OrderedDict`, `tuple` and `list`

```python
>>> Sequential(Linear(3, 5), Linear(5, 2).state_dict()

OrderedDict([('0.weight',
              tensor([[ 0.4512, -0.1306, -0.0298],
                      [ 0.2635,  0.5573, -0.0610],
                      [-0.5151, -0.2372, -0.2422],
                      [-0.1536, -0.5279, -0.2040],
                      [-0.3668,  0.4431,  0.3355]])),
             ('0.bias', tensor([-0.0857,  0.3729,  0.0929,  0.3936, -0.1156])),
             ('1.weight',
              tensor([[-0.2642,  0.0674,  0.2640,  0.2075, -0.4029],
                      [-0.3660, -0.0844,  0.1445,  0.4002, -0.3518]])),
             ('1.bias', tensor([ 0.1969, -0.0726]))])
```

Reply:
Good idea, added in 7dfb1fa.

Anyway, notice that having a `Dict[str, torch.Tensor]` was already tested, so here the only difference is that we add a test for `OrderedDict` instead of `Dict`.

---

> It seems to use `OrderedDict`, `tuple` and `list`

As a side note, I think that what is `repr`esented as 2-ples `(key, value)` is just the way `OrderedDict` behaves, but I am not sure there are really `tuples` stored inside as `keys` or `values`.

Reply:
Good point!